grantnav/frontend/templatetags/graphs.py

```diff
@@ -64,19 +65,28 @@ def award_date_graph(context):
 
         x = []
         y = []
+        early_total = 0
         for bucket in award_year_buckets:
+            if bucket['key'] < YEAR_CUT_OFF:
+                early_total += bucket['doc_count']
             if bucket['key'] > YEAR_CUT_OFF:
                 x.append(bucket['key_as_string'])
                 y.append(bucket['doc_count'])
 
         x, y = (list(sync) for sync in zip(*sorted(zip(x, y))))
+        x.insert(0, f'{context["results"]["aggregations"]["earliest_grant"]["hits"]["hits"][0]["_source"]["awardDate"][:4]}+')
+        y.insert(0, early_total)
+
+        x2 = deepcopy(x)
```

Comment:
Can we call this something more descriptive like  `x_with_aggregated_first` or something along those lines